### ISSUE TITLE
can not autoplay on mobile device.

### ISSUE THREAD

**Issue by hVenus** [MichaelMammoliti/jAudio.js#1]
I use htc m8, jaudio can not autoplay in chrome.

**Comment by MichaelMammoliti** [comment]
You don't need this and autoplay is working fine.

Plugin.init() calls Plugin.changeTrack(). Plugin.changeTrack() checks whether the "autoPlay" property is truthy.

Audio.js v0.2, line 170:
`if(self.currentState === "play" || self.settings.autoPlay) self.play();`

**Comment by daowa** [comment]
line 160: self.stop()
commented out this line and jaudio can work on mobile device

**Comment by sapbmw** [comment]
@daowa  You are right. If commented out this line 160 and jaudio can auto play in IOS mobile device.

line 160:
self.stop();   change to ==> //self.stop();

but, it can't work in Android mobile device, who can  find the bug and fix it, it is very useful audio webpage.

SAPBMW.
